docs/gateway.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 5
+---
+
+# Trust Gateway
+
+## Purpose
+
```

Comment:
The Trust Gateway is designed as an open-source implementation of a trust network infrastructure for health credentialing.